JASP-Engine/JASP/R/binomialtestbayesian.R

```diff
@@ -310,12 +310,14 @@ BinomialTestBayesian <- function(jaspResults, dataset = NULL, options, ...) {
   dfPointsPP <- .dfPointsPP(a=options$priorA, b=options$priorB, hyp = hyp, theta0 = options$testValue, counts = counts, n = n)
   xName <- expression(paste("Population proportion ", theta))
   
+  hypForPlots <- .binomHypothesisForPlots(hyp)
+  
   if (!options$plotPriorAndPosteriorAdditionalInfo)
     p <- JASPgraphs::PlotPriorAndPosterior(dfLines = dfLinesPP, dfPoints = dfPointsPP, xName = xName, bfSubscripts = bfSubscripts)
```

Comment:
Does this call require any changes?

Reply:
yes because for the jaspGraphs function, hypothesis is one of `c("equal", "smaller", "greater")` whereas in the code above hyp was one of `c("two.sided", "less", "greater")` 
Using this with bfType and then removing bfSubscripts fixes the subscripting error, since the jaspGraphs function can make the subscripts using these anyways

Reply:
I meant `p <- JASPgraphs::PlotPriorAndPosterior(dfLines = dfLinesPP, dfPoints = dfPointsPP, xName = xName, bfSubscripts = bfSubscripts)`